fvm/src/kernel/default.rs

```diff
@@ -376,6 +376,12 @@ where
     fn msg_value_received(&self) -> TokenAmount {
         self.value_received.clone()
     }
+
+    fn msg_validate(&self, msg: Message, sig: Signature) -> Result<GasSpec> {
+        self.msg_caller();
+
+        todo!()
+    }
```

Comment:
I don't understand what this method would be for. Maybe add some docs? What are you thinking?

Reply:
leftover from some thoughts that I abandoned yesterday.